tokio/src/task/local.rs

```diff
@@ -327,12 +330,24 @@ const MAX_TASKS_PER_TICK: usize = 61;
 /// How often it check the remote queue first.
 const REMOTE_FIRST_INTERVAL: u8 = 31;
 
+/// Context guard for LocalSet
+#[allow(missing_debug_implementations)]
+pub struct LocalEnterGuard(Option<Rc<Context>>);
```

Comment:
Please derive Debug instead.

Reply:
The Context has deep structure that does not derive debug, in order to do that, need to add that to all types deep into context

Reply:
Okay. You can add an impl like this instead, then:
```Rust
impl fmt::Debug for LocalEnterGuard {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        f.debug_struct("LocalEnterGuard").finish()
    }
}
```

Reply:
Thanks! Will fix it